### PR TITLE
Fix incorrect spelling and path

### PR DESCRIPTION
**Pull request by linjicong**
Fix incorrect spelling and path
About_DolphinScheduler.md
upgrade.md

### CODE REVIEW

File: site_config/docs2-0-0.js [hunk 1]
```diff
@@ -283,7 +283,7 @@ export default {
         children: [
           {
             title: '简介',
-            link: '/zh-cn/docs/About_DolphinScheduler/About_DolphinScheduler.html',
+            link: '/zh-cn/docs/2.0.1/user_doc/About_DolphinScheduler/About_DolphinScheduler.html',
```
Comment (zhongjiajie) [inline site_config/docs2-0-0.js:286]: I fond out we still have this file in https://github.com/apache/dolphinscheduler-website/blob/5f4d4a913d0909f36a33c776c1d9cef3adb1ba11/docs/zh-cn/About_DolphinScheduler/About_DolphinScheduler.md could you remove the file. and add it to each verison doc?

Comment (zhongjiajie) [inline site_config/docs2-0-0.js:286]:
```suggestion
            link: '/zh-cn/docs/2.0.0/user_doc/About_DolphinScheduler/About_DolphinScheduler.html',
```

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

**Comment by linjicong** [comment]
I think i need more time to fix all dead links in old version docs.because it cannot pass Dead Link Checker.

**Comment by zhongjiajie** [comment]
> I think I need more time to fix all dead links in old version docs. because it cannot pass the Dead Link Checker.

Our dead link checker has some error when you add a new page to it, you could see in https://github.com/apache/dolphinscheduler/issues/7642

I fond out error cause 

```log
  [✖] https://dolphinscheduler.apache.org/zh-cn/docs/1.3.8/user_doc/system-manual.html → Status: 404
1638
Error: Process completed with exit code 1.
```

And the page you just add it in this PR, so I think is good and we should do nothing about it

File: site_config/docs2-0-2.js [hunk 2]
```diff
@@ -279,7 +279,7 @@ export default {
         children: [
           {
             title: '简介',
-            link: '/zh-cn/docs/About_DolphinScheduler/About_DolphinScheduler.html',
+            link: '/zh-cn/docs/2.0.2/user_doc/About_DolphinScheduler/About_DolphinScheduler.html',
```
Comment (zhongjiajie) [inline site_config/docs2-0-2.js:282]: We should also change dev document

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

**Review by zhongjiajie — APPROVED** [review]
LGTM, Thanks for your contribution

**Pull request merged**